### PR TITLE
feat: sort imports

### PR DESCRIPTION
**Pull request by scottdover**
**Summary**
We recently made some changes in other repositories (studio next / studio commons) to sort our imports in a more predictable way. This copies our common config over.

**Testing**
 - [x] Ran `npm run format`
 - [x] Clicked through extension to make sure things worked as expected

### CODE REVIEW

File: prettier.config.js [hunk 1]
```diff
@@ -1,2 +1,22 @@
-// take default
-module.exports = {};
+module.exports = {
+  plugins: ["@trivago/prettier-plugin-sort-imports"],
+  importOrder: [
+    "^react(.*)$",
```
[the remark this answers is not in the document]
Comment (scottdover) [reply prettier.config.js:4]: Not a bad idea. I'll include it.

File: prettier.config.js [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+    "^@(.*)$",
```
[the remark this answers is not in the document]
Comment (scottdover) [reply prettier.config.js:5]: 🤷 For this one, I'm basically copying the config we use for studio next, studio commons, and api first dev portal. That said, you're right in that `@` isn't really going to appear elsewhere

File: prettier.config.js [hunk 3]
```diff
@@ -1,2 +1,22 @@
...
+    "<THIRD_PARTY_MODULES>",
+    "^[./].*(?<!.(css|scss|css.json|properties))$",
+    // ".(properties)$",
+    // ".(css|scss|css.json)$",
```
[the remark this answers is not in the document]
Comment (scottdover) [reply prettier.config.js:9]: My mistake. I'll clean this up shortly.

File: client/src/browser/extension.ts [hunk 4]
```diff
@@ -1,9 +1,7 @@
 // Copyright © 2022, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-
```
[the remark this answers is not in the document]
Comment (scottdover) [reply client/src/browser/extension.ts:3]: I don't think it's feasible to preserve it with this plugin. I _also_ like that empty line

File: prettier.config.js [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
   importOrder: [
+    "^vscode(.*)$",
     "^react(.*)$",
     "^@(.*)$",
     "<THIRD_PARTY_MODULES>",
-    "^[./].*(?<!.(css|scss|css.json|properties))$",
-    // ".(properties)$",
-    // ".(css|scss|css.json)$",
+    "^[./].*(?<!.(css))$",
+    ".(css)$",
```
Comment (scnwwu) [inline prettier.config.js:9]: nit: shouldn't it be `\.` otherwise it matches any character?

**Review by scnwwu — APPROVED** [review]
OK, overall good to me. Just a minor comment.

**Review by scnwwu — APPROVED** [review]

**Comment by scottdover** [comment]
There's nothing to test with this as it's just an infrastructure change.

**Pull request merged**